internal/consul/kvstore.go

```diff
@@ -0,0 +1,22 @@
+package consul
+
+// The Trento Agent is periodically updating data structures in the Consul Key-Value Store
+// to be consumed by the Trento Web Console
+//
+// At some point in time this structure needs to become a versioned protocol between the
+// Trento Web Console and the Trento Agent
+
+// For now, until we tighten the structure around versioning, this file is a start
+// that records constants that need to be in sync or have compat behavior between
+// Console and Agent
+
+// ### add landscape and system to the name
+const KV_CLUSTERS_PATH string = "trento/v0/clusters"
```

Comment:
Just to comment on this, I started changing to camel case the constants, as you started because it looks like it is more golang thing. Just a comment hehe. 
We should definitely pick one definitive option

Reply:
well, don't really care which style we use. I changed it to all upper case based on previous review-feedback.

Reply:
let's stick with `camelCase`.

https://stackoverflow.com/a/22688926/1488104

We should add this as a note in the contributing documentation.

Reply:
added a `CONTRIBUTING.md` and referenced Effective Go in there.